### PR TITLE
Add url support for Church Center

### PR DESCRIPTION
**Pull request by seven1m**
The main commit is 7599b2afde89f19011ab747582f382e39b1c62b8, but I took the liberty of cleaning up some other stuff while I was mucking around in there.

### CODE REVIEW

File: lib/pco/url/church_center.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+module PCO
+  class URL
+    class ChurchCenter < URL
+      def initialize(app_name: "church-center", path: nil, query: nil, encrypt_query_params: false, domain: nil)
```
Comment (danott) [inline lib/pco/url/church_center.rb]: What do you think about `s/domain/subdomain/` within this model? `subdomain` has been the name used across the existing `ChurchCenter` related repos.

**Comment by danott** [comment]
:+1:

**Comment by bensie** [comment]
:+1:

**Comment by seven1m** [comment]
@danott I added subdomain support in ef414ade1eb84759ff50b714fc8149d4d091def4.

**Pull request merged**